src/repos/WeeklyDebriefRepo.ts

```diff
@@ -0,0 +1,56 @@
+import { DocumentType } from '@typegoose/typegoose';
+import { User, UserModel } from '../entities/User';
+import WeeklyDebrief from '../entities/WeeklyDebrief';
+import { ArticleModel } from '../entities/Article';
+import { DAYS_IN_WEEK } from '../common/constants'
+
+const getExpirationDate = (creationDate: Date): Date => {
+  creationDate.setDate(creationDate.getDate() + DAYS_IN_WEEK);
+  return creationDate;
+};
+
+const createWeeklyDebrief = async (
+  user: DocumentType<User>,
+  creationDate: Date,
+  expirationDate: Date,
+): Promise<User> => {
+  const articleAggregate = ArticleModel.aggregate();
+  const { uuid } = user;
+  const weeklyDebrief = Object.assign(new WeeklyDebrief(), {
+    uuid,
+    creationDate,
+    expirationDate,
+    numShoutouts: user.numShoutouts,
+    numBookmarkedArticles: user.numBookmarkedArticles,
+    readArticles: user.readArticles.slice(0, 2),
```

Comment:
you're sure this wont give an error right..?

Reply:
yes